backend/qemu.pm

```diff
@@ -34,11 +37,16 @@ use constant LONG_MAX => (~0 >> 1);
 # in a separate dir.
 use constant VM_SNAPSHOTS_DIR => 'vm-snapshots';
 
+our @EXPORT_OK = 'QEMU_BACKEND_VERSION';
+
 sub new ($class) {
     my $self = $class->SUPER::new;
     $self->{pidfilename} = 'qemu.pid';
     $self->{proc} = OpenQA::Qemu::Proc->new();
     $self->{proc}->_process->pidfile($self->{pidfilename});
+
+    bmwqemu::fctinfo('QEMU_BACKEND_VERSION: ' . QEMU_BACKEND_VERSION);
+
```

Comment:
See my comment about the commit in general. But more specifically leave out blank lines please

Reply:
I don't see your link.

Sure I remove the blank lines. (OT: Spaces help readability, most of C projects asks for space after variables and before return. Also looking to some perl tools for Debian (old perl hackers), most ot them separate with blank line variables from code.)

Reply:
yes but I don't see why that one statement needs to be separated by blank lines while others in before don't so be to consistent just don't add blank lines between statements.